pyleecan/Methods/Mesh/MeshSolution/save_mesh.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+from scipy.io import savemat
+
+
+def save_mesh(self, save_path):
+    """Save mesh to a .mat file
+
+    Parameters
+    ----------
+    self : MeshSolution
+        an MeshSolution object
+    save_path : str
+        full path including folder, name and extension of the file to save if save_path is not None
+
+    """
+
```

Comment:
check isdir => raise Exception
Check path ends with ".mat" if no extension add it, else raise exception